src/Directory.Build.props

```diff
@@ -31,10 +31,10 @@
     <EnableNETAnalyzers>True</EnableNETAnalyzers>
     <AnalysisLevel>8.0.0</AnalysisLevel>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors Condition="('$(Configuration)' == 'Release') Or ('$(Configuration)' == 'Release462') Or ('$(Configuration)' == 'Release70') Or ('$(Configuration)' == 'ReleaseAll')">true</TreatWarningsAsErrors>
```

Comment:
why any changes here?

Reply:
I want to be able to run a debug build with warnings
It was like that in the past but recent changes create a "regresssion"
This change isn't related w/ this PR